includes/abstracts/abstract-wc-data.php

```diff
@@ -278,6 +294,14 @@ public function get_meta_data() {
 	 * @return mixed
 	 */
 	public function get_meta( $key = '', $single = true, $context = 'view' ) {
+		if ( ! $this->validate_meta_key( $key ) ) {
```

Comment:
Given the context, would this method be better named `is_internal_meta_key` or similar?